gov_uk_dashboards/components/plotly/paragraph.py

```diff
@@ -11,14 +13,16 @@ class ParagraphSizes(str, Enum):
     SMALL = "govuk-body-s"
 
 
-def paragraph(text: str, size: ParagraphSizes = ParagraphSizes.DEFAULT) -> html.P:
-    """Create a <p> Html component with the text provided"""
+def paragraph(
+    children: Union[str, list], size: ParagraphSizes = ParagraphSizes.DEFAULT
+) -> html.P:
+    """Create a <p> Html component with the children provided"""
```

Comment:
Not a blocker, but I do like the fuller Google style DocString format used elsewhere for example [gov_uk_dashboards/components/plotly/table.py#L15-L35](https://github.com/communitiesuk/pkg_gov_uk_dashboards/blob/main/gov_uk_dashboards/components/plotly/table.py#L15-L35).

I wonder if there is a linting option or similar which can encourage documenting arguments inside of DocStrings 🤔